web/src/components/storage/dasd/DASDTable.tsx

```diff
@@ -325,7 +371,7 @@ export default function DASDTable() {
         </ToolbarContent>
       </Toolbar>
 
-      <Page.Section>
+      <Page.Section aria-label="DASDs table section">
```

Comment:
Please, mark the text for translation. Also, I'd go for a `DASDs devices and actions` or similar. Without the section word, which is kind of useless.

Anyway, I guess you have adding this because a complain of the Page.Section component, which I've realize must be improved to allow sections without such an attribute when accepting `aria-labelledby` or, maybe, change the `role` of the section neither is given `title` nor `aria-label`.

What we can do by now is just ignore the `console.error` you saw in either, the developer tools or the test output.